apps/condo/domains/user/schema/ConfirmPhoneAction.test.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * Generated by `createschema user.ConfirmPhoneAction 'phone:Text;token:Text;smsCode:Integer;smsCodeRequestedAt:DateTimeUtc;smsCodeExpiresAt:DateTimeUtc;retries:Integer;isPhoneVerified:Checkbox;requestedAt:DateTimeUtc;expiresAt:DateTimeUtc;completedAt:DateTimeUtc;'`
+ */
+
+const { makeClient, makeLoggedInAdminClient, makeLoggedInClient } = require('@core/keystone/test.utils')
+const { 
+    createTestPhone,
+    createTestUser,
+    ConfirmPhoneAction, 
+    createTestConfirmPhoneAction, 
+    updateTestConfirmPhoneAction,
+} = require('@condo/domains/user/utils/testSchema')
+const { 
+    CONFIRM_PHONE_ACTION_EXPIRED,
+    CONFIRM_PHONE_SMS_CODE_EXPIRED,
+    CONFIRM_PHONE_SMS_CODE_VERIFICATION_FAILED,
+    CONFIRM_PHONE_SMS_CODE_MAX_RETRIES_REACHED,
```

Comment:
We also need to have max phone verify SMS to a phone per day

Reply:
Will add